Instead of hardcoding a wrong prototype for a libcap function, include
the correct header.

Index: argus-3.0.0.rc.34/argus/ArgusSource.c
===================================================================
--- argus-3.0.0.rc.34.orig/argus/ArgusSource.c	2010-02-09 22:57:21.000000000 +0100
+++ argus-3.0.0.rc.34/argus/ArgusSource.c	2010-02-09 22:57:29.000000000 +0100
@@ -53,6 +53,7 @@
 #define PPP_HDRLEN      4       /* length of PPP header */
 #endif
 
+#include <pcap.h>
 
 void ArgusGetInterfaceStatus (struct ArgusSourceStruct *src);
 
Index: argus-3.0.0.rc.34/argus/ArgusSource.h
===================================================================
--- argus-3.0.0.rc.34.orig/argus/ArgusSource.h	2010-02-09 22:57:35.000000000 +0100
+++ argus-3.0.0.rc.34/argus/ArgusSource.h	2010-02-09 22:57:54.000000000 +0100
@@ -381,7 +381,6 @@
 
 int ArgusCreatePktFromFddi(const struct fddi_header *, struct ether_header *, int);
 
-extern char *bpf_image(struct bpf_insn *, int);
 
 
 #else /* defined(ArgusSource) */
